spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/system/configuration/ConfigImpl.java

```diff
@@ -86,9 +88,7 @@ public Long getLong(String key, Long defaultValue) {
     @Override
     public Config getConfig(String path) {
         String absolutePath = absolutePathOf(path);
-        var filteredEntries = entries.entrySet().stream()
-                .filter(entry -> entry.getKey().startsWith(absolutePath))
-                .collect(TO_MAP);
+        var filteredEntries = entries.entrySet().stream().filter(entry -> entry.getKey().startsWith(absolutePath)).collect(TO_MAP);
```

Comment:
Personally I prefer to avoid row longer than 120 characters.
In addition, stream fluent chains are way more readable with some indentation